format/index.md

```diff
@@ -988,11 +988,18 @@ COMMA, ",") list that refers to the name(s) of the fields to be returned.
 If a client requests a restricted set of [fields], an endpoint **MUST NOT**
 include additional [fields] in the response.
 
+The `[` and `]` characters in a `fields[TYPE]` parameter **MUST** be
+percent-encoded, per the URI formatting requirements
+[detailed in RFC 3986](http://tools.ietf.org/html/rfc3986#section-3.4).
+
```

Comment:
I think we can eliminate this normative addition and leave the non-normative notes to clarify usage.